app/models.py

```diff
@@ -2163,6 +2163,23 @@ class BroadcastStatusType(db.Model):
 
     STATUSES = [DRAFT, PENDING_APPROVAL, REJECTED, BROADCASTING, COMPLETED, CANCELLED, TECHNICAL_FAILURE]
 
+    EDITABLE_STATUSES = [DRAFT, PENDING_APPROVAL, REJECTED]
```

Comment:
I got a bit confused at first reading through this PR because `EDITABLE_STATUSES` sounds like the status of the broadcast message can be changed, when it's the message itself which can be changed. You could maybe update the name just to make it extra clear what this refers to - up to you though.

Reply:
Yeah, agreed that the name is a bit awkward.  On reflection I think it's probably good to contrast it to `LIVE_STATUSES` (as that's the key distinction), but `NON_LIVE_STATUSES` sounds a bit clunky - any suggestions?

Reply:
Maybe something like `EDITABLE_MESSAGE_STATUSES` to try and show it is a list of states where the message can be edited, not a list of states where the status can be edited. But it's hard to think of a good name, and I'm not sure that `EDITABLE_MESSAGE_STATUSES` is an improvement, so feel free to ignore the earlier comment.

Reply:
for now i changed them to `PRE_BROADCAST_STATUSES` as i think it's more useful to describe what they actually represent (messages that havent started broadcast yet), rather than how we intend to use the flag, which is a secondary effect